keylime/ima/ima.py

```diff
@@ -610,6 +615,30 @@ def canonicalize_allowlist(alist_bytes, checksum=""):
     return alist
 
 
+def ima_policy_db_contents(ima_policy_name: str, ima_policy: str, tpm_policy: bool = False) -> dict:
```

Comment:
Is tpm_policy just a bool (rather than a str)? It seems to be `text` in the DB table...

Reply:
It is `str`, right